src/mach.rs

```diff
@@ -787,13 +799,20 @@ fn build_relocations(segment: &mut SegmentBuilder, artifact: &Artifact, symtab:
                 // NB: we currently deduce the meaning of our relocation from from decls -> to decl relocations
                 // e.g., global static data references, are constructed from Data -> Data links
                 match (link.from.decl, link.to.decl) {
-                    (Decl::Defined(DefinedDecl::DebugSection { .. }), _) => {
+                    (Decl::Defined(DefinedDecl::Section(s)), _)
+                        if s.kind() == SectionKind::Debug => {
```

Comment:
The indentation here is weird. Need to run rustfmt?

Reply:
Yes :)